weatherApp.js

```diff
@@ -206,25 +216,27 @@ const createHourBoxes = (day) => {
 
 const renderDivlist = (htmlElements, nameOfList) => {
 	const divList = document.createElement("div");
-       htmlElements.forEach((element) =>{
-           divList.appendChild(element);
-       });
+	htmlElements.forEach((element) =>{
+	   divList.appendChild(element);
+	});
     divList.className = nameOfList;
     const list = document.getElementsByClassName(nameOfList)[0];
     list.parentNode.replaceChild(divList, list);
 }
 
 let fetchWeather = (city)=>{
-/*    const requestUrl = `https://api.openweathermap.org/data/2.5/forecast?q=${city}&APPID=26c42f8b8305b74545f93ce538ad2356`;*/
-	const requestUrl = `dupadupadupadupadupadupadupa`;
+    const requestUrl = `https://api.openweathermap.org/data/2.5/forecast?q=${city}&APPID=26c42f8b8305b74545f93ce538ad2356`;
```

Comment:
Weren't you supposed to migrate to different api?

Reply:
But I decided to stay with this :)